libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/lanes/CarLanesImageRenderer.kt

```diff
@@ -20,26 +22,30 @@ class CarLanesImageRenderer(
     private val carLaneIconRenderer = CarLaneIconRenderer(context)
     private val laneIconsApi = MapboxLaneIconsApi()
     private val carLaneIconMapper = CarLaneMapper()
+    private val cache = LruCache<Lane, CarLanesImage>(1)
```

Comment:
What's the purpose of a maxSize=1 here? 🤔

Reply:
The number of items held by the `LruCache`.

Reply:
Right but why not 5?

Reply:
`Lane` is actually a collection of multiple lane indicators. We want a new image rendered for every change to the lanes. I was confused but 1 makes sense